src/index.html

```diff
@@ -22,6 +22,345 @@
+          <button
+            class="card__Buy"
+            type="button"
+            data-qa="hover"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
+      <div class="product">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          />
+          <div class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__serial_number">Product code: 195434</div>
+          <div class="card__reviews">
+            <div class="reviews_stars">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <div class="reviews_text">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="price_text">Price:</div>
+            <div class="price_number">$2,199</div>
+          </div>
+          <button
+            class="card__Buy"
+            type="button"
+            data-qa="hover"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
+      <div class="product">
+        <div
+          class="card"
+          data-qa="card-hover"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          />
+          <div class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__serial_number">Product code: 195434</div>
+          <div class="card__reviews">
+            <div class="reviews_stars">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <div class="reviews_text">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="price_text">Price:</div>
+            <div class="price_number">$2,199</div>
+          </div>
+          <button
+            class="card__Buy"
+            type="button"
+            data-qa="hover"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
+      <div class="product">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          />
+          <div class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__serial_number">Product code: 195434</div>
+          <div class="card__reviews">
+            <div class="reviews_stars">
+              <div class="star"></div>
```

Comment:
```suggestion
            <div class="stars">
              <div class="stars__star"></div>
```
It should be a separate block

Reply:
done